firmware/pico/src/app/ads_bee.hh

```diff
@@ -122,29 +122,6 @@ class ADSBee {
      */
     void OnDemodComplete();
 
-    /**
-     * ISR triggered by SysTick interrupt. Used to wrap the MLAT counter.
-     */
-    void OnSysTickWrap();
-
-    /**
-     * Creates a composite timestamp using the current value of the SysTick timer (running at 125MHz) and the SysTick
-     * wrap counter to simulate a timer running at 48MHz (which matches the frequency of the preamble detector PIO).
-     * @param[in] num_bits Number of bits to mask the counter value to. Defaults to full resolution.
-     * @retval 48MHz counter value.
-     */
-    inline uint64_t GetMLAT48MHzCounts(uint16_t num_bits = 64);
```

Comment:
The new functionality looks like it removes the num_bits optional parameter. This was previously used to mask the 48MHz counter to arbitrary bit lengths, and I think it has an instance of being used where we generate a 12MHz counter from the 48MHz counter. I think we probably need to keep the num_bits parameter.